scope.go

```diff
@@ -1230,19 +1235,19 @@ func (scope *Scope) removeForeignKey(field string, dest string) {
 	scope.Raw(fmt.Sprintf(query, scope.QuotedTableName(), scope.quoteIfPossible(keyName))).Exec()
 }
 
-func (scope *Scope) removeIndex(indexName string) {
-	scope.Dialect().RemoveIndex(scope.TableName(), indexName)
+func (scope *Scope) RemoveIndexContext(indexName string) {
```

Comment:
did we want to make this public?

this has the same signature and replaces the old method. Does it need to be a new method?

Reply:
Also think this is a find and replace mistake. Will undo